great_expectations/expectations/core/expect_column_values_to_be_null.py

```diff
@@ -220,9 +220,7 @@ def _validate(
         result_format = self.get_result_format(
             configuration=configuration, runtime_configuration=runtime_configuration
         )
-        mostly = self.get_success_kwargs().get(
-            "mostly", self.default_kwarg_values.get("mostly")
-        )
+        mostly = self.get_success_kwargs().get("mostly")
```

Comment:
Should this also use `self._get_default_value` to match the pattern of the other changes?

Reply:
Nope. AFAIK the previous pattern was redundant; `get_success_kwargs` calls `_get_default_value` (and previously reached into the dict)